Tasks/AzureResourceGroupDeployment/operations/ResourceGroup.ts

```diff
@@ -366,6 +366,8 @@ export class ResourceGroup {
             parameters = this.updateOverrideParameters(template, parameters);
         }
 
+        parameters = this.sanitizeParameters(parameters);
```

Comment:
This needs to be called in PublicURL deployment as well

Reply:
done